app/lib/methods/enterpriseModules.js

```diff
@@ -33,7 +33,7 @@ export function getEnterpriseModules() {
 	return new Promise(async(resolve) => {
 		try {
 			const { version: serverVersion, server: serverId } = reduxStore.getState().server;
-			if (serverVersion && gte(serverVersion, '3.1.0')) {
+			if (serverVersion && isServerVersionGreaterThan(serverVersion, '3.1.0')) {
```

Comment:
@gerzonc One more thing here.
We're repeating ourselves on `serverVersion && ` in all calls.
Do you agree that should be inside of the util functions? :)